src/user/components/nav_bar.js

```diff
@@ -1,21 +1,40 @@
 import React from 'react'
+import ReactDOM from 'react-dom'
 import { map, zip } from 'ramda'
 import { Navbar, Nav, NavItem } from 'react-bootstrap'
 
-export default () =>
-  <Navbar>
-    <Navbar.Toggle />
-    <Navbar.Collapse>
-      <Nav>
-        {map(item)(zip(routes, text))}
-      </Nav>
-    </Navbar.Collapse>
-  </Navbar>
+class NavigationBar extends React.Component {
+  componentDidMount () {
+    const navBar = ReactDOM.findDOMNode(this.refs.navigation_bar)
+    const collapsibleNav = navBar.querySelector('div.navbar-collapse')
+    const btnToggle = navBar.querySelector('button.navbar-toggle')
 
+    navBar.addEventListener('click', (e) => {
+      if (e.target.tagName !== 'A' ||
+          e.target.classList.contains('dropdown-toggle') ||
+          ! collapsibleNav.classList.contains('in')) return
+      btnToggle.click()
+    }, false)
```

Comment:
Apparently it is for most modern browsers but is not in ancient ones. Leave it in or take it out?